dist/index.html

```diff
@@ -27,7 +24,7 @@ <h1>Today's To Do</h1><span><i class="fas fa-sync-alt"></i></span>
            
         </div>
         <div class="button">
-            <button id="button">clear all completed</button>
+            <button id="button">Clear completed</button>
```

Comment:
- [ ] [Required] Kindly, Add button type for each button as it required by HTML [Webhint](https://webhint.io/) linter.